.github/workflows/rock-pull-request.yaml

```diff
@@ -0,0 +1,10 @@
+name: Pull Request
+# quality checks to make sure the change won't break things
+
+on:
+  workflow_call:
```

Comment:
Should we add "on pull request"? Or have `rock-update.yaml` call it?
Atm this seems unused.

Reply:
This is a reusable workflow; `on: pull_request` is what the importing workflow on each rock repo will have. Otherwise, it would trigger on PRs to this repo :)